components/PostCardList/PostCardListItem.tsx

```diff
@@ -0,0 +1,227 @@
+import * as React from "react";
+import styled from "styled-components";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import Post from "../../models/Post";
+import calculateFinalPot from "../../utilities/calculateFinalPot";
+import getPositionByPlayerAndIndex from "../../utilities/getPositionByPlayerAndIndex";
+import {
+  getRelativeDateString,
+  getRelativeShortDateString
+} from "../../utilities/getRelativeDateString";
+import getStringWithSIMetricSuffix from "../../utilities/getStringWithSIMetricSuffix";
+import Card from "../Card";
+import { ThumbsUpIcon } from "../Icon";
+import PlayingCard from "../PlayingCard";
+import PostTypeContext, { PostType } from "./PostTypeContext";
+
+interface Props extends React.Attributes {
+  post: Post;
+  onClick?: React.MouseEventHandler<HTMLElement>;
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+export default function PostCardListItem({ post, ...props }: Props) {
+  const postType = React.useContext(PostTypeContext);
+
+  return (
+    <Root {...props}>
+      <HeroHand>
+        <HeroHandBackGround />
+        <HandCard
```

Comment:
"hand card" sounds weird because hand means a set of cards itself. How about this instead?

```suggestion
        <HeroCard
```